src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+// Actions
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+// Reducer
+const reducer = (state = [], action = {}) => {
+  let newState;
+  switch (action.type) {
+    case ADD_BOOK:
+      newState = [...state];
+      return newState.push(action.payload);
+
+    case REMOVE_BOOK:
+      newState = state.filter((book) => book.id !== action.payload.id);
+      return newState.push(action.payload);
+
+    default: return state;
+  }
```

Comment:
- [ ] [OPTIONAL] I suggest  instead of using a variable and assigning a state to that variable you can do it directly like: 
```JAVASCRIPT
// FOR ADD_BOOK
return [...state, action.payload]
// FOR REMOVE_BOOK:
return [state.filter((book) => book.id !== action.payload.id]
```
as you can see it's cleaner